Android/app/src/main/java/utils/Utils.java

```diff
@@ -65,12 +66,12 @@ public static String decryt(String d) throws UnsupportedEncodingException {
     }
 
     public static boolean isNetworkConnected(Context context) {
-        ConnectivityManager cm =
-                (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
no need of changes in this file also

Reply:
This is my IDE doing auto adjust when i changed line wrap. Fixed this!.